backend/tests/test_profile.py

```
@@ -0,0 +1,167 @@
+    yield
+    
+    # Teardown: Drop tables and remove upload directory
+    Base.metadata.drop_all(bind=engine)
+    if os.path.exists(upload_dir):
+        shutil.rmtree(upload_dir)
+
+# ----------------------
+# 🔧 Fixture for test user
+# ----------------------
+@pytest.fixture
+def test_user():
```

Ruff (0.8.2)

Missing return type annotation for public function `test_user`

(ANN201)